Dashboard/app/js/templates/navReports/export-reports.handlebars

```diff
@@ -30,50 +31,56 @@
         document.getElementById(exportName).style.display = "block";
         evt.currentTarget.className += " active";
        }
-    </script> 
+    </script>      
```

Comment:
Please remove the whitespace...

Reply:
there was no white space on my end, locally